.gitignore

```diff
@@ -3,17 +3,18 @@
 # Logs
 *.log
 
-# Extras
-extra/lgit.rb
-
-# Notes
-notes/go.notes
-TODO.notes
```

Comment:
Could we please leave TODO.notes and TODO.md here as I am too lazy to move these files from my root project directory :)

Otherwise LGTM